server/controller/src/main/java/ai/starwhale/mlops/api/RuntimeApi.java

```diff
@@ -327,7 +327,9 @@ ResponseEntity<ResponseMessage<BuildImageResult>> buildRuntimeImage(
             @Parameter(in = ParameterIn.PATH, required = true, schema = @Schema())
             @PathVariable("runtimeUrl") String runtimeUrl,
             @Parameter(in = ParameterIn.PATH, required = true, schema = @Schema())
-            @PathVariable("versionUrl") String versionUrl);
+            @PathVariable("versionUrl") String versionUrl,
+            @Parameter(description = "user defined running configurations such environment variables")
+            @RequestBody(required = false) String runConfig);
```

Comment:
Would it be better to use the structure? In this way, the schema can be displayed directly on the swagger

Reply:
done